scss/tech.scss

```diff
@@ -52,3 +51,27 @@
   grid-column: colum 2;
   grid-row: row 2;
 }
+
+@media only screen and (max-width: 1024px) {
+  .grid-container {
+    grid-template-columns: 100%;
+
+    .tech-container {
+      flex-direction: column;
+      grid-column-start: 1;
+      grid-row-end: 3;
+      justify-self: center;
+    }
+
+    .tech {
+      border-top: 2px $color-green solid;
+    }
+  }
+
+  .tech-main {
+    border-right: 2px $color-white solid;
```

Comment:
Are you hiding this border? `border: none` or `border: 0` is the way, you can use `border-right` in case you  want to keep bottom border